### PR TITLE
MATE 1.28 Bug fixes

### PR DESCRIPTION
**Pull request by immolo**
This is a selection of bug fixes for MATE 1.28 to make this ready to become stable in Gentoo

Please check all the boxes that apply:

- [X] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [X] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [X] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [X] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: app-editors/pluma-plugins/pluma-plugins-1.28.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Copyright 2021-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (mm1ke) [inline app-editors/pluma-plugins/pluma-plugins-1.28.0.ebuild:4]:
```suggestion
EAPI=8
```

Comment (immolo) [reply app-editors/pluma-plugins/pluma-plugins-1.28.0.ebuild:4]: Thanks, missed that one.

**Review by juippis — APPROVED** [review]
There are some minor issues with the ebuilds but since they've been there, let's not focus on them in this PR.

**Comment by immolo** [comment]
> There are some minor issues with the ebuilds but since they've been there, let's not focus on them in this PR.

Thanks  for merging. All issues should be in the 1.26 branch only from running pkgcheck however if I did miss something it will picked up as I start dropping old ebuilds in the future.

**Pull request closed without merging**